bundle/manifests/cert-manager.clusterserviceversion.yaml

```diff
@@ -0,0 +1,152 @@
+      mediatype: image/png
+  install:
+    spec:
+      deployments:
+        - name: cert-manager-operator
+          spec:
+            replicas: 1
+            selector:
+              matchLabels:
+                name: cert-manager-operator
+            template:
+              metadata:
+                labels:
+                  name: cert-manager-operator
+              spec:
+                containers:
+                  - command:
+                      - /usr/bin/cert-manager-operator
+                    args:
+                      - start
+                    env:
+                      - name: WATCH_NAMESPACE
+                        valueFrom:
+                          fieldRef:
+                            fieldPath: metadata.annotations['olm.targetNamespaces']
+                      - name: POD_NAME
+                        valueFrom:
+                          fieldRef:
+                            fieldPath: metadata.name
+                      - name: OPERATOR_NAME
+                        value: cert-manager-operator
+                    image: docker.io/slaskawi/cert-manager-operator:latest
+                    imagePullPolicy: Always
+                    name: cert-manager-operator
+                    resources: { }
+                serviceAccountName: cert-manager-operator
+      permissions:
```

Comment:
I'll be extending this list very shortly. It will be much, much (, much...) longer.